calculation/seistech_calc/seistech_calc/disagg/DisaggResult.py

```diff
@@ -288,13 +269,25 @@ def get_save_dir(im: IM, exceedance: float = None, im_value: float = None):
 
     @classmethod
     def load(cls, data_dir: Path):
-        metadata, site_info, fault_disagg, ds_disagg, mean_values = cls._load_data(
-            data_dir
+        with open(data_dir / cls.METADATA_FN, "r") as f:
+            metadata = json.load(f)
+
+        mean_values = (
+            pd.read_csv(data_dir / cls.MEAN_VALUES_FN, index_col=0).squeeze()
+            if (data_dir / cls.MEAN_VALUES_FN).exists()
+            else None
```

Comment:
Not sure if this really needs to change? I see that it's only used here currently, but in future if we did want to implement BranchDisaggResult load wouldn't we still want that base function.

Reply:
Yes, and if we do then it should be changed back. But we haven't needed to implement a BranchDisaggResult load function at this point, so it probably won't ever be needed. So then the base function just becomes extra spaghetti code imo

Reply:
Ok fair